wemake_python_styleguide/visitors/ast/complexity/counts.py

```diff
@@ -72,6 +77,31 @@ def _check_decorators_count(self, node: ModuleMembers) -> None:
                 ),
             )
 
+    def _check_method_decorators(self, node: ModuleMembers) -> None:
+        parent_is_class = isinstance(get_parent(node), ast.ClassDef)
+
+        # Check whether filtered decorators are method decorators
+        is_method_decorator = []
+        for decorator in node.decorator_list:
+            if isinstance(decorator, ast.Name):
+                is_method_decorator.append(
+                    decorator.id in {'classmethod', 'staticmethod', 'property'},
+                )
+
+        # Check for presence of method deecorators
+        has_method_decorators = (
+            reduce(
+                lambda boolean1, boolean2: boolean1 or boolean2,
+                is_method_decorator,
+                0,
```

Comment:
It should be `False`, not `0`

Reply:
Thank you so much for all the feedback and comments! I agree with all of them and really appreciate it! I have a question with changing `0` to `False` because it seems to be giving me this error when i run `flake8 .`:
```
./wemake_python_styleguide/visitors/ast/complexity/counts.py

  96:17    WPS425 Found boolean non-keyword argument: False
  False,
  ^
```
Should I still change it to `False` even though it would mean the `flake8 .` would not pass?

Thanks!

Reply:
Try `initializer=False`

Reply:
I'm getting this error when I changed it to
```
# Check for presence of method deecorators
has_method_decorators = (
    reduce(
        lambda boolean1, boolean2: operator.or_(boolean1, boolean2),
        method_decorators,
        initializer=False,
    )
)
```

```
TypeError: reduce() takes no keyword arguments
```

Reply:
I managed to figure out that storing `false` in a variable would not raise the error. What do you think of this:

```
default_param = False
has_method_decorators = (
    reduce(
        lambda boolean1, boolean2: operator.or_(boolean1, boolean2),
        method_decorators,
        default_param,
    )
)
```

Reply:
I guess, `# noqa` is just better in this case.

Reply:
Thanks a lot! (Oh, I really dislike this builtin functions which do not allow kw-args!)